.github/workflows/_charm-quality-checks.yaml

```diff
@@ -292,3 +292,24 @@ jobs:
         with:
           timeout-minutes: 30
           limit-access-to-actor: true
+
+      - name: Show logs
+        run: | 
+          if test -d ./.logs; then
+            for filename in ./.logs/*.txt; do
+                for ((i=0; i<=3; i++)); do
+                      echo "::group::{{ $filename }}"
+                      cat "$filename"
+                      echo "::endgroup::"
+                done
+            done
+          else
+            echo "./.logs not found; ensure you're flying with pytest-jubilant > 0.7"
```

Comment:
As we mentioned, logs could be dumped without `pytest-jubilant`, so I would remove that from the log line